recipes/testfixtures/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "4.10.0" %}
+{%set md5 = "64af471a82ab788c1f18e6c2b8bf0479" %}
+
+package:
+    name: testfixtures
+    version: {{ version }}
+
+source:
+    fn: testfixtures-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/t/testfixtures/testfixtures-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - testfixtures
+
+about:
+    home: http://pythonhosted.org/testfixtures/
```

Comment:
Seems this has been changed to ( https://github.com/Simplistix/testfixtures ) according to [PyPI]( https://pypi.python.org/pypi/testfixtures/4.10.0 ).